cupy/lib/polynomial.pyx

```diff
@@ -123,13 +126,12 @@ cdef class poly1d:
             return poly1d(self.coeffs * other)
         raise NotImplementedError
 
-    # TODO(Dahlia-Chehata): implement using polyadd
     def __add__(self, other):
```

Comment:
If a user-defined array object whose `__array_priority__` is greater than 90, its `__radd__` should be called. Could you fix like as [`cupy.ndarray.__add__` is implemented](https://github.com/cupy/cupy/blob/v7/cupy/core/core.pyx#L965), and add a test?

Reply:
Won't that affect `test_poly1d_add_poly1d_array` ?

Reply:
Yes. it affects.